CODING_STANDARD

```diff
@@ -171,6 +171,20 @@ Output:
   (status(), error(), debug(), etc)
 - Use '\n' instead of std::endl
 
+Unit tests:
+ - Unit tests are written using Catch: https://github.com/philsquared/Catch/
+ - For large classes:
+    - Each file should contain tests for one function in one class
+    - The file should be put inside unit/class/path/class_name/function_name.cpp
+ - For small classes:
+    - Each file should contain the tests for the class
```

Comment:
Create a separate file for each class that contains the tests for all methods of that class